.gitignore

```diff
@@ -1,4 +1,6 @@
 .snowpack
+.vscode/
```

Comment:
I actually changed to vscode to be inline with how you guys work, and some of my extensions there started editing files in .vscode, I can can exclude the specific file that was edited by my instance instead of the entire folder, on it